From c7ad24edca81bd463235ebe8399bac243bae177b Mon Sep 17 00:00:00 2001
From: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
Date: Tue, 30 May 2023 17:59:47 +0300
Subject: [PATCH 1/2] [FIO toimx] of: enable using OF_DYNAMIC without
 OF_UNITTEST

Since commit [1] there are i.MX drivers which use the OF_DYNAMIC
feature. Let the option OF_DYNAMIC enable independently of
OF_UNITTEST.

[1]
commit a553d46954894 ("MLK-17275-1 drm/bridge: adv7511: Add support for OF_DYNAMIC")

Upstream-Status: Pending
Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---

 drivers/of/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/of/Kconfig b/drivers/of/Kconfig
index 80b5fd44ab1c7..89204c537b06c 100644
--- a/drivers/of/Kconfig
+++ b/drivers/of/Kconfig
@@ -55,7 +55,7 @@ config OF_KOBJ
 # Hardly any platforms need this.  It is safe to select, but only do so if you
 # need it.
 config OF_DYNAMIC
-	bool "Support for dynamic device trees" if OF_UNITTEST
+	bool "Support for dynamic device trees"
 	select OF_KOBJ
 	help
 	  On some platforms, the device tree can be manipulated at runtime.
-- 
2.40.1

